.fuel-core/configs/chainConfig.json

```diff
@@ -74,226 +74,227 @@
-              "units_per_gas": 325
+              "base": 3,
+              "units_per_gas": 385
             }
           },
           "mcpi": {
             "LightOperation": {
-              "base": 8,
-              "units_per_gas": 511
+              "base": 7,
+              "units_per_gas": 585
             }
           },
           "meq": {
             "LightOperation": {
-              "base": 3,
-              "units_per_gas": 940
+              "base": 2,
+              "units_per_gas": 1234
             }
           },
           "retd_contract": {
             "LightOperation": {
-              "base": 305,
-              "units_per_gas": 4
+              "base": 227,
+              "units_per_gas": 5
             }
           },
           "s256": {
             "LightOperation": {
-              "base": 31,
-              "units_per_gas": 4
+              "base": 25,
+              "units_per_gas": 5
             }
           },
           "scwq": {
             "HeavyOperation": {
-              "base": 16346,
-              "gas_per_unit": 17163
+              "base": 5666,
+              "gas_per_unit": 6628
             }
           },
           "smo": {
             "LightOperation": {
-              "base": 40860,
-              "units_per_gas": 2
+              "base": 14635,
+              "units_per_gas": 3
             }
           },
           "srwq": {
             "HeavyOperation": {
-              "base": 187,
-              "gas_per_unit": 179
+              "base": 245,
+              "gas_per_unit": 243
             }
           },
           "swwq": {
             "HeavyOperation": {
-              "base": 17046,
-              "gas_per_unit": 16232
+              "base": 5661,
+              "gas_per_unit": 5776
             }
           },
           "contract_root": {
             "LightOperation": {
-              "base": 31,
-              "units_per_gas": 2
+              "base": 24,
+              "units_per_gas": 3
             }
           },
           "state_root": {
             "HeavyOperation": {
-              "base": 236,
-              "gas_per_unit": 122
+              "base": 189,
+              "gas_per_unit": 96
             }
           },
           "new_storage_per_byte": 63,
           "vm_initialization": {
             "LightOperation": {
-              "base": 3957,
-              "units_per_gas": 48
+              "base": 3127,
+              "units_per_gas": 61
             }
           }
         }
       },
-      "base_asset_id": "0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07",
+      "base_asset_id": "f8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07",
       "block_gas_limit": 30000000,
-      "privileged_address": "0000000000000000000000000000000000000000000000000000000000000000"
+      "block_transaction_size_limit": 260096,
+      "privileged_address": "eb42c979046d091f4f524e1efd8d4ffad92629b9a506b268a511861b0dbf2366"
```

Comment:
where is this `privileged_address` from? I assume you override it somewhere in the tests to allow for block production?

Reply:
Yeah we do override it some tests. I'll reset it. I think I got it from the chain config repo